framework/src/org/checkerframework/framework/type/TypeFromExpressionVisitor.java

```diff
@@ -362,14 +370,16 @@ public AnnotatedTypeMirror visitWildcard(WildcardTree node,
     @Override
     public AnnotatedTypeMirror visitPrimitiveType(PrimitiveTypeTree node,
                                                   AnnotatedTypeFactory f) {
-        // for e.g. "int.class"
+        // Node is the primitive type when primitive class literals are visited.
```

Comment:
Can this ever be reached? In "int.class" I would expect that the visitMemberSelect above is visited first. There the primitive is handled by f.getAnnotatedType(elt) and the primitive type itself is not visited.
So is there a different case where this gets used?